src/runtime.rs

```diff
@@ -26,30 +26,34 @@ struct Stuff {
     outcome_error_fn: PyObject,
 }
 
-static STUFF: GILOnceCell<PyResult<Stuff>> = GILOnceCell::new();
+static STUFF: GILOnceCell<Stuff> = GILOnceCell::new();
 
-fn get_stuff(py: Python<'_>) -> PyResult<&Stuff> {
-    let res = STUFF.get_or_init(py, || {
-        let trio_lowlevel = py.import("trio")?.getattr("lowlevel")?;
-        let outcome = py.import("outcome")?;
-        Ok(Stuff {
-            tokio_runtime: tokio::runtime::Runtime::new()?,
-            trio_current_trio_token_fn: trio_lowlevel.getattr("current_trio_token")?.into_py(py),
-            trio_reschedule_fn: trio_lowlevel.getattr("reschedule")?.into_py(py),
-            trio_current_task_fn: trio_lowlevel.getattr("current_task")?.into_py(py),
-            trio_wait_task_rescheduled: trio_lowlevel.getattr("wait_task_rescheduled")?.into_py(py),
-            trio_abort_succeeded: trio_lowlevel
-                .getattr("Abort")?
-                .getattr("SUCCEEDED")?
-                .into_py(py),
-            outcome_value_fn: outcome.getattr("Value")?.into_py(py),
-            outcome_error_fn: outcome.getattr("Error")?.into_py(py),
-        })
-    });
-    match res {
-        Ok(stuff) => Ok(stuff),
-        Err(err) => Err(err.clone_ref(py)),
-    }
+fn get_stuff(py: Python<'_>) -> &Stuff {
+    STUFF.get_or_init(py, || {
+        // We accept unwrap because it runs once and it avoids the Result everytime
```

Comment:
Outdated comment ? maybe you refer to the `expect` below

Reply:
expect is unwrap with a custom message, so it's ok

Reply:
Can you move done the comment to the `res.expect`

Reply:
Done